pkg/noderesourcetopology/cache/overreserve.go

```diff
@@ -208,8 +210,8 @@ func (ov *OverReserve) Resync() {
 
 	var nrtUpdates []*topologyv1alpha2.NodeResourceTopology
 	for _, nodeName := range nodeNames {
-		nrtCandidate, err := ov.nrtLister.Get(nodeName)
-		if err != nil {
+		nrtCandidate := &topologyv1alpha2.NodeResourceTopology{}
+		if err := ov.client.Get(context.Background(), types.NamespacedName{Name: nodeName}, nrtCandidate); err != nil {
```

Comment:
I wonder if we should use the same context for all the `Get`s - and I think we should (and we should also make it cancellable as further extension). But this change is better served in a followup PR.

Reply:
yup, other PRs also have some leftovers. The general idea would be use context wherever you can:

1. the chained calls in PreFitler/Filter can use the context that is provided by scheduler framework
2. for calls outside PreFilter/Filter/..., we should instantiate one at the start of main() (or make some change in scheduler framework to expose it), and pass it to all plugins

Reply:
created an issue to track this https://github.com/kubernetes-sigs/scheduler-plugins/issues/664